src/nni_manager/config/kubeflow/pytorchjob-crd-v1.json

```diff
@@ -0,0 +1,17 @@
+{
+    "kind": "CustomResourceDefinition",
+    "spec": {
+        "scope": "Namespaced",
+        "version": "v1",
+        "group": "kubeflow.org",
+        "names": {
+            "kind": "PyTorchJob",
+            "plural": "pytorchjobs",
+            "singular": "pytorchjob"
+        }
+    },
+    "apiVersion": "apiextensions.k8s.io/v1beta1",
```

Comment:
v1beta1?

Reply:
> https://github.com/kubeflow/tf-operator/blob/master/examples/crd/crd-v1.yaml
They use it so here.

Reply:
Already have test it. it's working :)

Reply:
Checked again, https://github.com/kubeflow/pytorch-operator/blob/master/examples/mnist/v1/pytorch_job_mnist_nccl.yaml using `apiVersion: "kubeflow.org/v1"` instead of `"apiVersion": "apiextensions.k8s.io/v1beta1"`. 

Will get back after doing more tests.

Reply:
Thanks for the test, better to use `apiVersion: "kubeflow.org/v1"` since it's more clear.

Reply:
Tests have been done. Turns out both "`kubeflow.org/v1`" and "`apiextensions.k8s.io/v1beta1`" are working well. As you mentioned,  better to use apiVersion: "`kubeflow.org/v1`" since it's more clear. The code has been updated, please kindly have a look